monitor/entrypoint

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
```

Comment:
I think it only needs to wait for RABBITMQ

Reply:
make you mean?